<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncoderSettingsDialog</class>
 <widget class="QDialog" name="EncoderSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Image Encoder Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QSpinBox" name="imageQuality">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://doc.qt.io/qt-5/qpixmap.html#save&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Quality&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="defaultImageQuality">
        <property name="text">
         <string>Format default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Video Encoder Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QTabWidget" name="videoTabs">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="hTab">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <attribute name="title">
          <string notr="true">h264/h265</string>
         </attribute>
         <layout class="QFormLayout" name="verticalLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://trac.ffmpeg.org/wiki/Encode/H.264#crf&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Preset&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="profileBox">
            <property name="currentText">
             <string notr="true">ultrafast</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">ultrafast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">superfast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">veryfast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">faster</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">fast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">slow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">slower</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">veryslow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">placebo</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://trac.ffmpeg.org/wiki/Encode/H.264#crf&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;CRF&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="crf">
            <property name="maximum">
             <number>51</number>
            </property>
            <property name="value">
             <number>23</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="vpTab">
         <attribute name="title">
          <string notr="true">VP9</string>
         </attribute>
         <layout class="QFormLayout" name="verticalLayout1">
          <item row="0" column="1">
           <widget class="QCheckBox" name="vp9Lossless">
            <property name="text">
             <string>Lossless (not recommended)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="gifTab">
         <attribute name="title">
          <string notr="true">GIF</string>
         </attribute>
         <layout class="QFormLayout" name="verticalLayout2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>TODO: Find whatever configuration GIF can have in ffmpeg's libav</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Bitrate</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="bitrate">
        <property name="suffix">
         <string notr="true">kbps</string>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="value">
         <double>400.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="toolTip">
         <string>The number of pictures in a group of pictures, or 0 for intra only </string>
        </property>
        <property name="text">
         <string>GOP size</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="gopSize">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EncoderSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EncoderSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
